### ISSUE TITLE
SSR Support

### ISSUE THREAD

**Issue by iluvsoup** [zerodevx/svelte-toast#56]
Currently this package doesn't seem to support SSR because of it's use of `document`

Not that important but it would be nice to have

I'm using SvelteKit

**Comment by zerodevx** [comment]
Thanks for the bug report. This is introduced in `v0.8` - FWIW it's *supposed* to work since `document` is only used in `onMount()` and should only run client-side... but I *think* I know what's wrong. I'll submit a fix soon.

**Comment by zerodevx** [comment]
@iluvsoup - I published a fix in the `@next` tag. Could you help test and see if the error shows again? 🙏  

You can install with:

```
$ npm i @zerodevx/svelte-toast@next
```

**Comment by iluvsoup** [comment]
Works great other than a visual bug whenever I refresh the page

https://user-images.githubusercontent.com/94307595/199830425-97c7df58-4330-4fb4-86c4-3e9c3f9c9faf.mp4

It happens on both chrome and firefox, could it be a svelte quirk?

Thanks for the quick reply by the way

**Comment by iluvsoup** [comment]
On second thought it might be an issue with my form submission, I'll look in to it

**Comment by iluvsoup** [comment]
Actually it might be something to do with the stores not clearing on refresh, the toasts don't ever seem to go away

![image](https://user-images.githubusercontent.com/94307595/199832447-d3ceccc0-2f71-4ba8-9c22-957882a9b1eb.png)
(the outputs of `console.log($toasts)`)

**Comment by zerodevx** [comment]
Wow, not sure what's going on to be honest. Is the error message gone? Also, could you try installing:

```
$ npm i @zerodevx/svelte-toast@0.7.2
```

and let me know if anything changed?

**Comment by iluvsoup** [comment]
Oh yeah the error message is gone, it's just the store don't seem to clear whenever I refresh which results in the giant wall of toasts

**Comment by zerodevx** [comment]
Does this still happen with `v0.7.2`? Just wanna isolate.

**Comment by iluvsoup** [comment]
Yes

**Comment by zerodevx** [comment]
Ah ok, I'm pretty sure we're facing 2 separate issues here. The `document is not defined` thing happens because while `onMount()` to guaranteed to only run client-side, it appears `onDestroy()` may be called during SSR - this will be fixed with #57.

On the other hand, the `wall of toasts` thing is likely related to your form submission code. If you can post a minimal reproduction, perhaps we can move this to a new issue and try to work it out from there.

For now, I'll publish a patch release which should mark this SSR issue as fixed.
